packages/taquito-local-forging/test/data/generic_contract.ts

```diff
@@ -1,8 +1,10 @@
+import { MichelsonValue } from "../../src/michelson/codec";
+
 export const genericStorage = {
   prim: 'Unit',
 };
 
-export const genericCode = (op: any) => [
+export const genericCode = (op: MichelsonValue) => [
```

Comment:
I think the type should be a string instead of MichelsonValue. It will be one of the values in opMapping: https://github.com/ecadlabs/taquito/blob/master/packages/taquito-local-forging/src/constants.ts#L45